lib/astarte/client/appengine/devices.ex

```diff
@@ -140,4 +140,22 @@ defmodule Astarte.Client.AppEngine.Devices do
       end
     end
   end
+
+  @doc """
+  Patches a writable property of a device.
+  """
+  def patch_device_writeable_property(%AppEngine{} = client, device_id, data) do
```

Comment:
I would rename this as `update_device_writeable_property`
The HTML `PATCH` verb is an implementation detail here.